### PR TITLE
Feature/pagination

### PR DESCRIPTION
**Pull request by mdobosz3**
Add pagination to rewards

### CODE REVIEW

File: spec/system/reward_spec.rb [hunk 1]
```diff
@@ -7,25 +7,51 @@
...
-
-    click_link 'Back'
-    expect(page).to have_content reward.title
-    expect(page).not_to have_content reward.description
-    expect(page).to have_content reward.price
+  context 'when the employee checks the rewards' do
+    let(:employee) { create(:employee) }
+    let!(:reward) { create(:reward) }
+
+    let!(:reward1) { create(:reward) }
```
Comment (ciejj) [inline spec/system/reward_spec.rb:14]: consider using: `FactoryBot.create_list(:reward, 10)`

File: app/controllers/rewards_controller.rb [hunk 2]
```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
 
 class RewardsController < ApplicationController
+  REWARDS_PER_PAGE = 10
+
   def index
-    render :index, locals: { rewards: Reward.all }
+    @page = params.fetch(:page, 0).to_i
+    @number_of_page = (Reward.count / REWARDS_PER_PAGE).ceil
```
Comment (ciejj) [inline app/controllers/rewards_controller.rb:8]: I suggest renaming it to:
```suggestion
    @number_of_pages = (Reward.count / REWARDS_PER_PAGE).ceil
```

**Review by ciejj — APPROVED** [review]
Well done - good work on < and << 👍

File: app/controllers/rewards_controller.rb (hunk 2 above)
Comment (mdobosz3) [reply app/controllers/rewards_controller.rb]: I corrected the nomenclature

File: spec/system/reward_spec.rb (hunk 1 above)
Comment (mdobosz3) [reply spec/system/reward_spec.rb]: I had a problem with it, but it worked and I used create_list :)

**Comment by oskarlakner** [comment]
🙌

**Pull request merged**